jujugui/static/gui/src/app/app.js

```diff
@@ -1240,13 +1240,14 @@ YUI.add('juju-gui', function(Y) {
     */
     _generateSocketUrl: function(callback) {
       this.jem = null;
-      if (this.get('jemUrl')) {
+      if (this.get('jemURL')) {
```

Comment:
I'll change that to jemURL as well, rather than changing charmstoreURL all over the place to maintain consistency.